src/parser/test/ParserTest.cpp

```diff
@@ -702,5 +702,45 @@ TEST(Parser, Annotation) {
     }
 }
 
+TEST(Parser, Agg) {
+    {
+        GQLParser parser;
+        std::string query = "ORDER BY $-.id";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
```

Comment:
Can the `ORDER BY` clause be used alone?

Reply:
i treat _ORDER BY_ as a sentence for now. It would not be a syntax error, but return nothing.

Reply:
Mark a TODO to explain.